src/user-guide/task-implementation/skip-mode.rst

```diff
@@ -0,0 +1,119 @@
+.. _task-run-modes.skip:
+
+Skip Mode
+=========
+
+.. versionadded:: 8.4.0
+
+Skip mode is designed as an aid to workflow control:
+
+* It allows creation of :term:`dummy tasks<dummy task>` as part of workflow
+  design.
+* It allows skipping of tasks in a running workflow using either:
+  * ``cylc broadcast -s 'run mode = skip'`` (for when it is ready to run)
+  * ``cylc set --out skip`` (to immediately skip)
+
+.. note::
+
+   Setting :cylc:conf:`[runtime][<namespace>]run mode=skip` in your ``flow.cylc``
+   will lead to ``cylc validate`` returning a warning::
+
+      WARNING - The following tasks are set to run in skip mode:
+        * example_task
+
+   This is designed to prevent users running a task in skip mode by mistake.
+   If you are using skip mode deliberately then this can be
+   ignored.
+
+Task Outputs
+^^^^^^^^^^^^
+
+Skip mode allows the user to specify which task outputs
+will be emitted using :cylc:conf:`[runtime][<namespace>][skip]outputs`.
+
+By default:
+
+* All required outputs will be generated.
+* ``succeeded`` will be generated even if success is :term:`optional <optional output>`.
+* If ``[skip]outputs`` is specified and does not include either ``succeeded``
+  or ``failed``, then ``succeeded`` will still be generated.
+
+The outputs submitted and started are always generated and do not
+need to be defined in outputs.
+
+Task Event Handlers
+^^^^^^^^^^^^^^^^^^^
+
+By default task event handlers are disabled by skip mode, but they
+can be enabled using
+:cylc:conf:`[runtime][<namespace>][skip]disable task event handlers`.
```

Comment:
Referencing long configurations can be a pain when writing the docs and full configuration paths might be overkill if the context of the writing is obvious.

FYI, you can shorten these absolute configs to relative ones in one of two ways...

1. We can use the `.. cylc:scope::` directive, all future `:cylc:conf:` references will be relative to the specified section. Remember to reset this later on in the file:

```rest
.. cylc:scope:: flow.cylc[runtime][<namespace>]

can be enabled by using :cylc:conf:`[skip]disable task event handlers`.

.. cylc:scope::
```

2. Use a `<target>`

```rest
:cylc:conf:`[skip]disable task event handlers <flow.cylc[runtime][<namespace>][skip]disable task event handlers>`

Reply:
Thanks. TIL - the former seems handy in this context.

Reply:
I couldn't get the `.. cylc:scope::` directive working as shown

Reply:
NVM

```diff
- .. cylc:scope::
+ .. cylc-scope::
```